yolox/core/trainer.py

```diff
@@ -276,6 +282,12 @@ def after_iter(self):
                         "train/lr": self.meter["lr"].latest
                     })
                     self.wandb_logger.log_metrics(metrics, step=self.progress_in_iter)
+                if self.args.logger=='mlflow':
+                    logs = {"train/" + k: v.latest for k, v in loss_meter.items()}
+                    logs.update({
```

Comment:
Is there any need to break the line here?

Reply:
I copied these two line from wandb logger, I think it is output of code beautifier, any removed the extra line.